groceries_app/views.py

```diff
@@ -10,5 +11,10 @@ class AboutView(TemplateView):
 class RecipesView(TemplateView):
     template_name = 'recipes/recipes.html'
 
-class PlannerView(TemplateView):
-    template_name = 'grocery_planner/planner.html'
\ No newline at end of file
+class MealChoiceView(TemplateView):
+    template_name = 'grocery_planner/gp1-meal-choice.html'
```

Comment:
What is the relationship between Grocery Planner and Meal choice feature?

Reply:
Meal choice would be the first page you come to when working through the grocery planner.